takwimu/models/dashboard.py

```diff
@@ -137,6 +144,8 @@ class IndicatorWidgetsBlock(blocks.StreamBlock):
             ('title', blocks.CharBlock(required=False)),
             ('hide_title', blocks.BooleanBlock(default=False, required=False)),
             ('body', blocks.RichTextBlock(required=False)),
+            ('sdg', blocks.ChoiceBlock(required=False, choices=sdg_choices,
```

Comment:
Seems like it's possible for a single indicator to belong to two different SDGs. Please [confirm](https://unstats.un.org/sdgs/indicators/indicators-list/) it doesn't affect Takwimu indicators.